packages/relay/src/lib/eth.ts

```diff
@@ -442,6 +464,10 @@ export class EthImpl implements Eth {
     const requestIdPrefix = formatRequestIdMessage(requestId);
     this.logger.trace(`${requestIdPrefix} estimateGas(transaction=${JSON.stringify(transaction)}, _blockParam=${_blockParam})`);
 
+    if ("data" in transaction){
+      this.counter.labels('eth_estimateGas', transaction.data.substring(0,10)).inc();
```

Comment:
nit: `eth_estimateGas` should be static value `EthImpl.ethEstimateGas`

Reply:
Done.